.travis.yml

```diff
@@ -6,7 +6,7 @@ before_install:
 env:
 # Get some of them configs
 - HYPERBAHN_REMOTE_TEST_CONFIG=
-- HYPERBAHN_REMOTE_TEST_CONFIG=test/config/with_lazy_relay.json
+- HYPERBAHN_REMOTE_TEST_CONFIG=test/config/without_lazy_relay.json
```

Comment:
good catch, we did change the default recently, please break this out into a separate PR